### PR TITLE
printf → print + intoduction in variable

### PR DESCRIPTION
**Pull request by inem**
1) printf doesn't make sense here

2) it's better to separate data from code

### CODE REVIEW

File: sociability_test.rb [hunk 1]
```diff
@@ -69,16 +69,18 @@
   "наконец, подумайте о своем здоровье – такой стиль жизни не проходит бесследно."
 }
 
+puts introduction_text
```
Comment (inem) [inline sociability_test.rb]: I think .....

Comment (izeal) [reply sociability_test.rb]: kek

File: sociability_test.rb [hunk 2]
```diff
@@ -69,16 +69,18 @@
...
 result = 0
 
 questions_array.each do |question|
   puts("\n#{question}\n")
-  printf("Ваш ответ: ")
+  print("Ваш ответ: ")
 
   answer = gets.chomp.downcase
 
   until answer == "да" || answer == "иногда" || answer == "нет"
```
Comment (izeal) [inline sociability_test.rb]: здесь лучше %w(да иногда нет).include? answer

**Comment by Cuurjol** [comment]
I have modified the program, correcting some mistakes and defects.

**Comment by Cuurjol** [comment]
I have modified the program, correcting some mistakes and defects.

**Pull request closed without merging**